code/procs/antagonist_procs.dm

```diff
@@ -1,30 +1,3 @@
-/client/proc/gearspawn_traitor()
```

Comment:
Don't we need this for omnitraitor and weird edge cases?

Reply:
Not to my knowledge; all instances of `gearspawn_traitor()` were removed as I came across them while refactoring.